styleguide/components/callouts/callouts.stories.mdx

```diff
@@ -23,7 +25,9 @@ import templateStandard from './_standard.html.haml';
   </Story>
 </Preview>
 
-## Example
+## Example callout
+
+This style is when an example needs to be displayed. Also, use it for quotes.
```

Comment:
"Also, use it for quotes". Interesting, do you have an example of this? I'd expect quotes to be their own design pattern but I might just be missing the context here.

Reply:
This was from [the guidance that Zarlashta](https://docs.google.com/document/d/1xBPM_hrPuOyGoeiWYDTEXcNSvQjWGlup08tAIXG5f0w/edit#heading=h.b5hdw4glkauw), a content strategist, wrote a while ago. There was no example image but I [found this on the public site](https://www.citizensadvice.org.uk/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/#h-overdrafts).

Reply:
Ah interesting. In which case we probably don't want this guidance next to the example callout, as that component will always include the "Example" tag, which doesn't make sense for a quote. Perhaps suggest using the standard variant, or we need a separate design pattern for quotes.

Reply:
Good point. I don't know if we've considered another design for quotes, @soniaturcotte ? Let me ask Zarlashta this question in her doc and see what she thinks.